e2e/scientists_queries.yaml

```diff
@@ -48,6 +48,42 @@ queries:
           he published a paper on general relativity."
           - null
 
+  - query: relat-star-Physikalische-real-star-scientists-from-ulm
+    type: text
+    sparql: |
+      SELECT * WHERE {
+          ?x <is-a> <Scientist> .
+          ?x <Place_of_birth> <Ulm> .
+          ?t ql:contains-entity ?x .
+          ?t ql:contains-word "relat* Physikalische rela*"
+      }
+    checks:
+      - num_cols: 5
+      - selected: [ "?x", "?ql_textscore_t", "?t", "?ql_matchingword_t_relat", "?ql_matchingword_t_rela" ]
+      - contains_row:
+          - "<Albert_Einstein>"
+          - null
+          - null
```

Comment:
What does the "null" actually mean here? Are those not checked, or are they reaqlly empty?
And is this a test that checks your most recent column bugs?

Reply:
Those are just not checked. One is the textscore, which is not checked because it is not important here. And the other is the text itself, that I tried to test but I couldn't get it to run. I suspect that there was a problem with the escaping characters of the quotes, that the text contains. So I just skipped the column because I just wanted to test the matching word columns anyways.
And yes this tests would have failed with my most recent bug.